### PR TITLE
introduction to development with hydra

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

**Comment by SimonKraus** [comment]
I would put all pages into the existing "Build" category.
Any concerns?

**Comment by vgantchev** [comment]
great initiative! i can take it over from here if @enthusiastmartin doesn't mind?

**Comment by jak-pan** [comment]
> great initiative! i can take it over from here if @enthusiastmartin doesn't mind?

Can you have a once-over for grammar and readability? I'll have a look too we'd like to merge this ASAP.

File: docs/dev_primitives.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+Current type: u32
+
+**Balance**
+
+Current type: u128
+
+**Price**
+
+Current type: U64F64
```
Comment (jak-pan) [inline docs/dev_primitives.md:27]: Not anymore

Comment (enthusiastmartin) [reply docs/dev_primitives.md]: Correct. Updated.

**Comment by vgantchev** [comment]
some polishing https://github.com/galacticcouncil/HydraDX-docs/pull/99

**Review by vgantchev — APPROVED** [review]

**Review by jak-pan — APPROVED** [review]

**Pull request merged**